projects/chaos/test/test_sequencer_fsm.c

```diff
@@ -287,6 +302,9 @@ void test_sequencer_fsm_interrupted_awaiting_error(void) {
       } else if (prev_event.id == CHAOS_EVENT_OPEN_RELAY) {
         present_event.id = CHAOS_EVENT_RELAY_OPENED;
         present_event.data = prev_event.data;
+      } else if (prev_event.id == CHAOS_EVENT_DELAY_MS) {
```

Comment:
Should this whole block just be made into a helper function?

Reply:
Done, although the assertions and a number of other qualities change so I can't make the whole thing a helper.